client_core/src/com/mygdx/game/client_core/model/GameState.java

```diff
@@ -3,14 +3,17 @@
 import com.artemis.ComponentMapper;
 import com.artemis.World;
 import com.badlogic.gdx.utils.IntArray;
+import com.mygdx.game.client_core.ecs.component.Movable;
 import com.mygdx.game.client_core.ecs.component.Score;
 import com.mygdx.game.core.ecs.component.Coordinates;
 import com.mygdx.game.core.ecs.component.SubField;
 import lombok.extern.java.Log;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
```

Comment:
tutaj też nieużywane importy - nie możesz przejechać przez cały kod narzędziem intellij do usuwania takich rzeczy?

Reply:
no muszę tak zrobić, czego używasz do tego?

Reply:
no gdzieś w intellij jest takie ustawienie, żeby przy commitach usuwać importy, ale też można przejechać, przez cały projekt i zmienić rzeczy jak odstępy czy coś

musisz poszukać, bo to jest wbudowane w intellij i pewnie znajdziesz w google